src/receive_imf.rs

```diff
@@ -253,27 +239,8 @@ pub(crate) async fn receive_imf_inner(
     } else {
         replace_msg_id = if rfc724_mid_orig == rfc724_mid {
             None
-        } else if let Some((old_msg_id, old_ts_sent)) =
```

Comment:
This part looks unrelated because looking at encrypted `timestamp_sent` is fine, and does this mean that duplicates will be never deleted, even if the message is deleted by the user?

Reply:
When user manually deletes a message, all messages with the same rfc724_mid are deleted. See `message::delete_msg_ex`.

Reply:
The code in `imap` is being removed, but it was just an optimization removing duplicates before decryption and parsing. A short comment won't make harm.

But i think that it's better to IMAP-delete not the duplicate, but the existing message instead, this way we don't affect receiving re-sent messages by other devices.